### PR TITLE
adds styletts2 as voice designer model

### PR DESCRIPTION
**Pull request by kasisnu**

### CODE REVIEW

File: crates/service/web/storyteller_web/src/http_server/endpoints/voice_designer/inference/enqueue_tts_request.rs [hunk 1]
```diff
@@ -189,7 +189,7 @@ pub async fn enqueue_tts_request(
     let query_result = insert_generic_inference_job(InsertGenericInferenceArgs {
         uuid_idempotency_token: &request.uuid_idempotency_token,
         inference_category: InferenceCategory::TextToSpeech,
-        maybe_model_type: Some(InferenceModelType::VallEX), // NB: Model is static during inference
+        maybe_model_type: Some(InferenceModelType::StyleTTS2), // NB: Model is static during inference
```
Comment (kasisnu) [inline crates/service/web/storyteller_web/src/http_server/endpoints/voice_designer/inference/enqueue_tts_request.rs:192]: We could keep vallex here if we want to delay switching

File: crates/service/job/inference_job/src/job/job_types/tts/styletts2/model_downloaders.rs [hunk 2]
```diff
@@ -0,0 +1,87 @@
+use crate::util::model_downloader::ModelDownloader;
+
+pub struct StyleTTS2Downloaders {
+    // pub styletts2: StyleTTS2ModelMapping,
+}
+
+crate::impl_model_downloader!(
```
Comment (ArEnSc) [inline crates/service/job/inference_job/src/job/job_types/tts/styletts2/model_downloaders.rs:7]: Not sure if we need these mappings at all this seems to be vall-e-x related

File: crates/service/job/inference_job/src/job/job_types/tts/styletts2/process_job_create_voice.rs [hunk 3]
```diff
@@ -0,0 +1,349 @@
...
+use mysql_queries::queries::voice_designer::voices::create_voice::CreateVoiceArgs;
+use tokens::tokens::users::UserToken;
+
+use crate::job;
+use crate::job::job_loop::job_success_result::JobSuccessResult;
+use crate::job::job_loop::job_success_result::ResultEntity;
+use crate::job::job_loop::process_single_job_error::ProcessSingleJobError;
+use crate::job::job_types::tts::styletts2::process_job::StyleTTS2ProcessJobArgs;
+
+const BUCKET_FILE_PREFIX_CREATE: &str = "fakeyou_";
```
Comment (ArEnSc) [inline crates/service/job/inference_job/src/job/job_types/tts/styletts2/process_job_create_voice.rs:32]: We will want to use a file descriptor for this that contains the prefix and suffix

File: crates/service/job/inference_job/src/job/job_types/tts/styletts2/model_downloaders.rs (hunk 2 above)
Comment (kasisnu) [reply crates/service/job/inference_job/src/job/job_types/tts/styletts2/model_downloaders.rs:7]: Oh yeah this can probably be taken out in cleanup

**Pull request merged**